struct-files.list

```diff
@@ -0,0 +1,58 @@
+compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Base.hs
```

Comment:
What is the purpose of this file ?

Reply:
It's the haskell definition of the DAML-LF AST